samples/edgehog_app/README.md

```diff
@@ -39,61 +110,85 @@ Where `<DEVICE_ID>` is the device ID of the device you would like to use in the
 is the hostname for your Astarte instance, `<REALM_NAME>` is the name of your testing realm and
 `<CREDENTIAL_SECRET>` is the credential secret obtained through the manual registration.
 
-### Configuration for fully functional Astarte
+In addition, the file `ca_certificates.h` should be modified, placing in the `ca_certificate_root`
+array a valid CA certificate in the PEM format.
+
+### Configuration for local Astarte/Edgehog
```

Comment:
These two section are optional i would denote it in a list in the section `## Configuring the device`, something along the lines of:
- If you are using a managed astarte instance ...
- If you are using a local astarte instance ...
The elements of the list should link the user to the section